### DISCUSSION TITLE
Third party cookies and gtag

### DISCUSSION THREAD

**Discussion by efimBistrov** [privacysandbox/privacy-sandbox-dev-support#173] [Phasing out third-party cookies] [answered]
When user set:
```
gtag("consent", "update", {
    ad_storage: "granted",
    analytics_storage: "granted"
});
```
as consent settings on our website, following warnings are thrown in google chrome console:
![Screenshot from 2023-10-23 11-47-04](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/assets/28001613/92e90436-b950-4352-a157-7b2fb6673b74)
and:
![Screenshot from 2023-10-23 11-47-27](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/assets/28001613/1c1b4874-0529-4ae4-8043-7aef45b844a3)

I've noticed that this happens only when ad_storage is "granted", analytics_storage can be set in any option and warnings are not shown.

What do we have to do to fix these warnings, if we have? Change some settings in google ads or google tag manager, something else? Or this issue will be fixed by google itself, since these are google cookies?

I've asked about this in google tag manager and google analytics help forums and stackoverflow but did not get any reply yet.

**Comment by efimBistrov** [comment]
I have asked this question in many places already and never received an answer. It seems that even Google themselves don’t yet know what will happen to these cookies.

**Comment by krgovind** [comment, marked as answer]
Hi @efimBistrov, Speaking on behalf of the Chrome team - our intention with issuing these warnings is to make web developers aware of upcoming third-party cookie phaseout in Chrome, and point them to resources so they may prepare their websites. 

I understand that some of these warnings are likely going to be related to subresources from third-parties (e.g. Google Tag Manager or Google Analytics in this case). I think reaching out on the forums for those products is the right approach; but my understanding is that those product teams are aware about the upcoming changes to Chrome, and will prepare their services accordingly.

I think you can ignore the warnings for this specific set of cookies. On our end, we're thinking about how we can reduce the noisiness of these warnings and make them more actionable. Any feedback on that front is appreciated!

**Reply by efimBistrov** [reply]
Big thanks for explanation. Finally I got answer, which i wanted😊

**Comment by samdutton** [comment]
Referenced in #229.
